Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -448,12 +449,12 @@ public final class AppDelegateViewModel: AppDelegateViewModelType, AppDelegateVi
       .skipNil()
 
     let projectRootLink = projectLink
-      .filter { _, subpage, _ in subpage == .root }
-      .map { _, _, vcs in vcs }
+      .filter { _, subpage, _, _ in subpage == .root }
+      .map { _, _, vcs, _ in vcs }
 
     let projectCommentsLink = projectLink
-      .filter { _, subpage, _ in subpage == .comments }
-      .map { project, _, vcs in vcs + [CommentsViewController.configuredWith(project: project, update: nil)] }
+      .filter { _, subpage, _, _ in subpage == .comments }
+      .map { project, _, vcs, _ in vcs + [CommentsViewController.configuredWith(project: project, update: nil)] }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kickstarter/ios-oss/blob/7af5d8cc3bc516409245ef9b3c8ec9c0b49c4f0c/Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift#L457"></span>Line should be 110 characters or less: currently 113 characters<br />
<code>line_length</code> <code>AppDelegateViewModel.swift:457</code></td>
    </tr>
  </tbody>
</table>